eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```diff
@@ -118,3 +121,11 @@ services:
             - [setFragmentPath, [%fragment.path%]]
             - [setSiteAccess, [@ezpublish.siteaccess]]
         abstract: true
+
+    ezpublish.converter.repository:
```

Comment:
And ideally, we should have 2 distinct converters here. One for location, one for content.